packages/calcite-components/src/components/tooltip/TooltipManager.ts

```diff
@@ -119,11 +119,7 @@ export default class TooltipManager {
     }
   };
 
-  private pointerDownHandler = (event: PointerEvent): void => {
-    if (!isPrimaryPointerButton(event)) {
-      return;
-    }
-
+  private clickHandler = (event: PointerEvent): void => {
```

Comment:
should we change to `MouseEvent` for consistency?

Reply:
May apply to other places. Doesn't really matter much since PointerEvent extends MouseEvent.

I also notice some other places using onPointerUp/onPointerDown or using PointerEvent typing for click events.

Reply:
Should we search and eliminate the util function `isPrimaryPointerButton`?

Reply:
> should we change to MouseEvent for consistency?

I think for our purposes, we can use `PointerEvent` as it matches the latest spec changes: https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#event_type. Worth noting that Firefox and Safari haven't updated yet (see [web platform tests](https://wpt.fyi/results/pointerevents?label=master&label=experimental&aligned&q=pointerevents%2F%20is_a_pointerevent)).

> I also notice some other places using onPointerUp/onPointerDown or using PointerEvent typing for click events.

I may have missed some, but I updated the places where `pointerdown` events were being treated as a `click`. I think `isPrimaryPointerButton` is still useful. E.g., color-picker's color field and sliders depend on the primary pointer's up and down events.

Reply:
If you spot the ones I missed, I can update them in a follow-up PR.